package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "schema-form-library",
```

Comment:
Should we rename this? This is really the non-public package so it probably doesn't matter too much, but nonetheless it seems like we should use the new name.

Reply:
Done!